p4-16/spec/grammar.mdk

```diff
@@ -877,6 +879,12 @@ actionRef
     ;
 // END:actionRef
 
+// BEGIN:entry
+entry
+    : optCONST optEntryPriority keysetExpression ':' actionRef optAnnotations ';'
+    ;
+// END:entry
+
```

Comment:
No change in definition of `entry` -- simply moving it earlier to match order that p4c defines them in.